### ISSUE TITLE
Option to fetch all list items in single collection

### ISSUE THREAD

**Issue by danejur** [joshstar/AniList-API-Docs#123]
So, in trying to implement custom lists in my app, I've realized that this task would be far less memory and cpu intensive if all list items (anime and manga) were returned in one monolithic collection while still maintaining their list_status property.

Would it be viable to allow a flag to be passed in with the user/{id}/animelist call to indicate that the API should return all list items under a single "list_items" collection, instead of the current "lists: list_name: {object}" structure? This may be a terrible idea, but it is a thought that I had. Perhaps this would best be dealt with if the API transitions to GraphQL.

Thoughts?

**Comment by joshstar** [comment]
Yeah this isn't something we can provide at the moment but should be pretty simple to do with Api v2 & GraphQL
